R/sass-dependencies.R

```diff
@@ -0,0 +1,58 @@
+#' Compile Sass to CSS using LibSass, then return an HTML Dependencies
+#'
+#' Calls `sass(input, ...)` and wraps the output into a [htmltools::htmlDependency()].
+#'
+#' @inheritParams sass
+#' @inheritParams htmltools::htmlDependency
+#' @param cache_key additional information to include in the `sass_hash()` call used to
+#' determine the output directory. This should include any information that could possibly
+#' influence the resulting CSS that isn't already captured by `input`. For example, if
+#' `input` contains something like `"@import sass_file.scss"` you may want to include the
+#' [file.mtime()] of `sass_file.scss` (or, perhaps, a [packageVersion()] if `sass_file.scss`
+#' is bundled with an R package).
+#' @param stylesheet Absolute file path(s) pointing to stylesheet(s) to include with the dependency
+#'
+#' @return returns a list of [htmltools::htmlDependency()] objects. The length
+#' of the list use 1 unless `input` contains [sass_layer()] object(s) that
+#' have `html_deps`.
+#'
+#' @export
+#' @seealso [sass()]
+#' @examples
+#' str(sass_html_dependencies(
```

Comment:
I think the example is better without `str()` -- that could confuse users. If you think it's important, you could save the result in a variable and then call `str()` on that.